documentation/guide-beanmapping.asciidoc

```diff
@@ -48,7 +61,7 @@ Important is that you configure all your custom datatypes as `<copy-by-reference
 </mapping>
 ----
 
-== Bean-Mapper Usage
+== Bean-Mapper Usage(Dozer/Orika)
```

Comment:
I would remove this suffix "(Dozer/Orika)". Further our conventions say not to use special characters like braces in section titles as docgen otherwise does not work properly.